lib/dice_bag/templates/databases/postgres.yml.dice

```diff
@@ -0,0 +1,18 @@
+<%= warning.as_yaml_comment %>
+
+<% environments = Rails.env.production? ? [:production] : [:development, :test] %>
+
+<% environments.each do |env| %>
+<%= env %>:
+  adapter: <%= configured[env].database_driver || 'postgres' %>
```

Comment:
This file is largely same as [the mysql config](https://github.com/mdsol/dice_bag/blob/69bbaae9a4877dbb1322f16f95c338e1bc9d418a/lib/dice_bag/templates/databases/mysql.yml.dice), modified adapter, db username, encoding, and port. Let me know if any different defaults we want to bake it. 🙇

Reply:
My eyes might be tired, but cannot find `port` 😎 :

`port: <%= configured[env].database_port || 5432 %>`

Reply:
Good catch..! 🙇 74ad031